controllers/login.js

```diff
@@ -0,0 +1,17 @@
+const { getLogin } = require('../models/loginDbModel')
+const { createNewJwt } = require('../services/authentication')
+
+exports.login = async (req, res, next) => {
+  try {
+    const { username, password } = req.body
+    const authResults = await getLogin(username, password)
```

Comment:
Please change the name from getLogin to verifyUser or authenticate. this function should return a boolean - true for correct user and pass and false if not